Content.Server/GameObjects/Components/Markers/MobSpawnerComponent.cs

```diff
@@ -0,0 +1,29 @@
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Random;
+using Robust.Shared.Serialization;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.ViewVariables;
+using System.Collections.Generic;
+
+namespace Content.Server.GameObjects.Components.Markers
+{
+    [RegisterComponent]
+    public class MobSpawnerComponent : Component, ISerializationHooks
+    {
+        [Dependency] private readonly IRobustRandom _robustRandom = default!;
```

Comment:
Given the entity's being spawned once it'd be better for this to be resolved in Initialize to reduce memory usage.

If you really wanted you could also have the component nuke itself as well.